client/src/components/Topics/TopicCard.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button } from '@mui/material';
+import React from 'react';
+import { Topic } from 'utils/apiRequests';
+import styles from './TopicCard.module.scss';
+import binary from './topic-images/binary-search-tree-nobg.png'
+import linked from './topic-images/linked-list-nobg.png';
+import graph from './topic-images/graph-nobg.png';
+
+interface Props {
+  topic: Topic;
+  index: number;
+}
+
+const images = [linked, binary, graph, binary];
```

Comment:
The 3rd one should be just be binary because it's an avl tree. When sorting algos gets merged in we can make an image for it